### PR TITLE
Update 'arcgis-js-api' for version 3.19

### PR DESCRIPTION
**Pull request by dasa**
Please fill in this template.

- [x] Make your PR against the `master` branch.
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://developers.arcgis.com/javascript/3/jshelp/whats_new.html
- [x] Increase the version number in the header if appropriate.

### CODE REVIEW

**Comment by dasa** [comment]
👍 

/cc @bsvensson

File: arcgis-js-api/index.d.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
-// Type definitions for ArcGIS API for JavaScript v3.18
-// Project: http://js.arcgis.com
-// Definitions by: Esri <http://www.esri.com>
+// Type definitions for ArcGIS API for JavaScript 3.19
+// Project: https://developers.arcgis.com/javascript/3/
+// Definitions by: Esri <https://github.com/Esri>
```
Comment (andy-ms) [inline arcgis-js-api/index.d.ts]: @Esri is an organization account. I don't think it gets notifications. Is there another account you could use?

Comment (dasa) [inline arcgis-js-api/index.d.ts]: notifications? I thought it was preferable to use a GH account vs. http://www.esri.com. Is that not the case?

Thanks

Comment (andy-ms) [inline arcgis-js-api/index.d.ts]: I meant vs. a user account. Although you could of course tell me if you get a notification when I mention @Esri.

**Pull request merged**